src/komponenter/krr-melding/krr-melding.test.tsx

```diff
@@ -1,10 +1,38 @@
 import * as React from 'react';
 import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
 import KrrMelding from './krr-melding';
+import { OppfolgingContext } from '../../ducks/oppfolging';
+import tekster from '../../tekster/tekster';
+
+const KRRProviders = function ({
+  oppfolgingContextProviderProps,
+}): ({ children }: { children: React.ReactChildren }) => React.ReactElement {
+  return ({ children }) => (
+      <OppfolgingContext.Provider value={oppfolgingContextProviderProps}>{children}</OppfolgingContext.Provider>
+  );
+};
 
 describe('Test av komponent', () => {
-  test('Rendrer komponent hvis reservasjon hos krr', () => {
-    render(<KrrMelding />)
-    expect(screen.getByText(/du er reservert mot digital kommunikasjon med det offentlige/i)).toBeTruthy()
+  test('Rendrer komponent hvis reservasjon hos krr', async () => {
```

Comment:
Funky async test. Hva er greia med det?

Reply:
Kjører ganske ofte async tester jeg :-)

Akkurat her er grunnen at `screen.queryByText` søket er async så da trenger jeg den for å kjøre `await`

Reply:
Litt tilleggsinfo, etter jeg nå har sjekket dokumentasjonen:
`screen.getByText` (som vi bruker ellers) returnerer elementene synkront, men vil kaste en feil dersom teksten ikke eksisterer. Det er dermed ikke mulig å bruke den til å teste negativer uten en assert på en `throws`.  `queryByText`  resolver derimot til en `null`-verdi hvis verdien ikke finnes, men asynkront.